internal/flags/helm_chart_source.go

```diff
@@ -64,13 +64,9 @@ func (s *HelmChartSource) Set(str string) error {
 }
 
 func (s *HelmChartSource) Type() string {
-	return "helmChartSource"
+	return strings.Join(supportedHelmChartSourceKinds, "|")
 }
 
 func (s *HelmChartSource) Description() string {
-	return fmt.Sprintf(
-		"source that contains the chart in the format '<kind>/<name>.<namespace>', "+
-			"where kind must be one of: (%s)",
-		strings.Join(supportedHelmChartSourceKinds, ", "),
-	)
+	return "source that contains the chart in the format '<kind>/<name>.<namespace>'"
```

Comment:
I'm not sure this fix make sense for this argument since the value has to match this format and the type is returning only the allowed kinds